src/python/pants/backend/python/util_rules/pex_cli.py

```diff
@@ -202,6 +208,15 @@ async def setup_pex_cli_process(
     )
 
 
+def maybe_log_pex_stderr(stderr: bytes, pex_verbosity: int) -> None:
+    """Forward Pex's stderr to a Pants logger if conditions are met."""
+    log_output = stderr.decode()
+    if log_output and pex_verbosity > 0:
+        logger.info("%s", log_output)
+    elif log_output and "PEXWarning:" in log_output:
+        logger.warning("%s", log_output)
```

Comment:
```suggestion
    if stderr and pex_verbosity > 0:
        logger.info("%s", stderr.decode())
    elif stderr and "PEXWarning:" in log_output:
        logger.warning("%s", stderr.decode())
```

This way, we only convert from bytes to str if required, similar to the old logic.

Reply:
Sorry if I'm reading this wrong, but I think this suggestion doesn't work because `log_output` will no longer be defined in the 2nd conditional.

FWIW What I've observed so far is that Pex's stderr is not noisy unless you enable verbosity, in which case we are going to decode anyway.

Reply:
LOL. True.